<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  
  <param name="use_cpu"           value="true" />

  <!-- Start yolov5 and ros wrapper -->
  <!-- <node pkg="yolov8_ros" type="yolo_v8.py" name="yolov8_ros" output="screen" > -->
  <!-- <group ns='object'>
  <node pkg="yolov8_ros" type="overlay.py" name="yolov8_ros" output="screen" >
    <param name="weight_path"       value="$(find yolov8_ros)/weights/"/>
    <param name="image_topic"       value="/camera/color/image_raw" />
    <param name="pub_topic"         value="/yolov8/BoundingBoxes" />
    <param name="camera_frame"       value="camera_color_frame"/>
    <param name="visualize"       value="true"/>
    <param name="conf"              value="0.3" />
  </node>
  </group> -->

  <group ns='hazmat'>
  <node pkg="yolov8_ros" type="yolo_v8.py" name="yolov8_ros" output="screen" >
    <param name="weight_path"       value="$(find yolov8_ros)/weights/"/>
    <!-- <param name="image_topic"       value="/camera/color/image_raw" /> -->
    <param name="image_topic"       value="/remote_robot/d435/camera/color/image_raw" /> 
    <param name="pub_topic"         value="/yolov8/BoundingBoxes" />
    <param name="camera_frame"       value="camera_color_frame"/>
    <param name="visualize"       value="true"/>
    <param name="conf"              value="0.3" />
  </node>
  </group>

</launch>
